src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String size;
+    private String type;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String size, String type) {
+        size = size;
```

Comment:
```suggestion
        this.size = size;
```
the same with other fields at the constructor

Reply:
Fixed.